ffi/dotnet/NowProtoTest/NowProtoTest.csproj

```diff
@@ -0,0 +1,27 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net8.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+
+    <IsPackable>false</IsPackable>
+    <IsTestProject>true</IsTestProject>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="coverlet.collector" Version="6.0.0" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.8.0" />
+    <PackageReference Include="MSTest.TestAdapter" Version="3.1.1" />
```

Comment:
Was it a good idea to use MSTest? Apparently, a lot of people hate MSTest, I picked it up as a first suggestion from VS 😅 If not, I could switch to NUnit or something different, but so far, no problems with MSTest

Reply:
No idea! In Devolutions Gateway repo I used Xunit:

https://github.com/Devolutions/devolutions-gateway/tree/master/utils/dotnet/Devolutions.Gateway.Utils.Tests

May be worth to uniformise, but not a big deal if it works!